firebase_admin/messaging.py

```diff
@@ -88,6 +99,56 @@ def send(message, dry_run=False, app=None):
     """
     return _get_messaging_service(app).send(message, dry_run)
 
+def send_all(messages, dry_run=False, app=None):
+    """Batch sends the given messages via Firebase Cloud Messaging (FCM).
```

Comment:
Lets change to `Sends the given list of messages via Firebase Cloud Messaging as a single batch.`